bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -58,6 +58,19 @@ spec:
 {{ toYaml . | indent 8 }}
       {{- end }}
 {{- include "cassandra.imagePullSecrets" . | indent 6 }}
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
```

Comment:
I presume we checked that we do not need an init container when persistence is disabled, right?

Reply:
That is, the container in the Dockerfile sets the right permissions already

Reply:
Yes, that should be the case. If it is not, we should fix the image